### PR TITLE
Fixes #399: support added for showing logs in PR status

### LINKED ISSUE #399: show the build logs in CI status
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

Yaydoc CI status redirecting to the yaydoc website
### Expected behavior

show the logs on  clicking the detail anchor tag

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

I have added support for showing build logs on PR status
## Related Issue

Close #399 
## Screenshots (if appropriate):
http://demo08.herokuapp.com/
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 please review

File: routes/ci.js [hunk 1]
```diff
@@ -204,7 +205,10 @@ router.post('/webhook', function(req, res, next) {
               if (req.body.action === "reopened" || req.body.action === "opened") {
                 if (repositoryData.PRStatus === true) {
                   var commitId = req.body.pull_request.head.sha;
-                  github.createStatus(commitId, req.body.repository.full_name, "pending", "Yaydoc is checking your build", repositoryData.accessToken, function(error, data) {
+                  var targetURL = `http://${process.env.HOSTNAME}/`;
+                  var name = `${req.body.pull_request.head.label.split(":")[0]}/${req.body.repository.full_name.split("/")[1]}`;
+                  console.log(name);
```
Comment (imujjwal96) [inline routes/ci.js]: Remove this?

Comment (sch00lb0y) [inline routes/ci.js]: removed

**Comment by sch00lb0y** [comment]
@imujjwal96 @pri22296 please review this

**Comment by imujjwal96** [comment]
image
Where is the metadata?

**Comment by imujjwal96** [comment]
Is there a need to create a separate model for PR  logs?

**Comment by sch00lb0y** [comment]
@imujjwal96 I'm not getting that info in webhook so I removed those details

**Comment by imujjwal96** [comment]
Travis shows the metadata for PRs
image

Take inspiration from them

**Comment by sch00lb0y** [comment]
@imujjwal96 I felt it will good if we have a separate scheme for PR so that it won't get confused with the build log. please check now. I have updated with meta values

**Comment by sch00lb0y** [comment]
@imujjwal96 @pri22296 please review this

**Comment by imujjwal96** [comment]
Please note the difference between the two images and make the necessary changes.

### Current Behavior:
image

### Expected Behavior:
image

**Comment by sch00lb0y** [comment]
@imujjwal96 updated.

**Comment by imujjwal96** [comment]
Please check again! You've'nt updated everything

**Comment by sch00lb0y** [comment]
@imujjwal96 I have updated the tabs as well now. please check

**Comment by imujjwal96** [comment]
Instead of showing a user's username, show their name.Same as we've done in Logs metadata. Add a single space after commit: and compare:

**Comment by sch00lb0y** [comment]
@imujjwal96 I have updated. please check Thanks!

**Pull request merged**